app/src/main/java/com/sdp13epfl2021/projmag/database/UserDataFirebase.kt

```diff
@@ -129,4 +137,43 @@ class UserDataFirebase(
                 .addOnFailureListener(onFailure)
         }
     }
+
+    override fun applyUnapply(
+        unapply: Boolean,
```

Comment:
This argument name does not match the interface and maybe is the opposite value ? (applied == !unapply)